components/net/src/oauth/github.rs

```diff
@@ -29,6 +30,7 @@ use config;
 use error::{Error, Result};
 
 const USER_AGENT: &'static str = "Habitat-Builder";
+const HTTP_TIMEOUT: u64 = 3_000;
```

Comment:
Should this be a value that we read from a configuration file?  Thinking of a private depot behind proxies, though 3 seconds does seem like a reasonably long time.